### ISSUE TITLE
Can't debug an application when a database on port 8080 is not called "default"

### ISSUE THREAD

**Issue by warpech** [Starcounter/Starcounter#3094]
I have a database "mydatabase" started on port 8080. 

When I open a project in Visual Studio and try to start and debug it, I get the following error:

> Severity	Code	Description	Project	File	Line
Error		This handler has already been registered. Can't register URI handler "GET /sc/db/{?}/{?}" on port 8080. This handler is already registered. (ScErrHandlerAlreadyRegistered (SCERR13003))

What happens is Starcounter creates a new database "default" and tries to start it on port 8080, which is already occupied by "mydatabase"

As a newbie I would not know what to do with this error message. Especially given the fact that "mydatabase" is the default name for a new database in Administrator Web UI.

![image](https://cloud.githubusercontent.com/assets/566463/11362834/736560ca-9296-11e5-9e5c-cff3e8a8dc27.png)

**Comment by per-samuelsson** [comment]
Yeah, this case will really confuse developers, and we need to get a top it. Its related also to recent #3078. We must consider this a bug IMO, because the combo of 1) high likelyhood of happening and 2) hard to understand how to get passed it for a new developer.

**Comment by per-samuelsson** [comment]
Also related: #2557.
